ttps/examples/steps/edit-step/edit-step.yaml

```diff
@@ -6,10 +6,11 @@ steps:
     inline: |
       set -e
 
+      echo -e "Target file pre-edit:"
       cat file-to-edit.txt
   - name: edit-target-file
     edit_file: "file-to-edit.txt"
-    backup_file: "my-backup.txt"
+    backup_file: "/tmp/my-backup.txt"
```

Comment:
I'll fix this first

Reply:
Sounds good. Yeah, basically it drops the backup in the CWD, which is different from where Forge expects it to be. Hence the absolute path.